client/dist/styles/overview.css

```diff
@@ -1,6 +1,7 @@
 #overviewProductInfo {
   display: grid;
   grid-template-columns: repeat(2, 700px);
+  grid-template-rows: 600px 200px;
```

Comment:
Try to use minmax() or max/min content to accomodate different size screens/adjusting window size